firefox-ios/Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -281,17 +281,6 @@ class AppSettingsTableViewController: SettingsTableViewController,
                                                           settingsDelegate: parentCoordinator))
         }
 
-        privacySettings.append(ClearPrivateDataSetting(settings: self, settingsDelegate: parentCoordinator))
-
-        privacySettings += [
-            BoolSetting(prefs: profile.prefs,
-                        theme: themeManager.currentTheme(for: windowUUID),
-                        prefKey: "settings.closePrivateTabs",
```

Comment:
@OrlaM I think there are a few other places with related code we can probably clean up

<img width="346" alt="Screenshot 2024-04-22 at 1 08 11 PM" src="https://github.com/mozilla-mobile/firefox-ios/assets/145381717/e97940b9-1e96-4685-a385-d5233049a4b8">

Reply:
Ok sounds good. Change LGTM